guides/source/asset_pipeline.md

```diff
@@ -21,12 +21,16 @@ What is the Asset Pipeline?
 The asset pipeline provides a framework to concatenate and minify or compress
 JavaScript and CSS assets. It also adds the ability to write these assets in
 other languages and pre-processors such as CoffeeScript, Sass and ERB.
+It allows assets in your application to be automatically combined with assets 
+from other gems. For example, jquery-rails includes a copy of jquery.js
+and enables AJAX features in Rails.
 
 The asset pipeline is technically no longer a core feature of Rails 4, it has
 been extracted out of the framework into the
 [sprockets-rails](https://github.com/rails/sprockets-rails) gem.
 
-The asset pipeline is enabled by default.
+The asset pipeline is enabled by default and is required for the normal operation 
```

Comment:
Not sure if this is correct - the asset pipeline can be removed from Rails -- it's not required for the normal operation of the framework.